ds-prep-python-tuples/main.py

```diff
@@ -0,0 +1,20 @@
+passenger = (12 , True , "Bonnell, Miss. Elizabeth" , "female" , 58)
+print(passenger)
+name = passenger[2]
+age = passenger[-1]
+print(name , age)
+survived_and_name = passenger[1:3]
+gender_and_age = passenger[3:]
+print(survived_and_name , gender_and_age)
+print("female" in passenger)
+is_female = "female" in passenger
+is_male = "male" in passenger
+print(is_female , is_male)
+def get_survival_info(passenger):
+  (id , survived , name , gender , age) = passenger
+  new_passenger = (age , gender , survived)
+  return new_passenger
+print(get_survival_info)
```

Comment:
Don't forget to call this at least once with `passenger` as an argument:

---

![image](https://user-images.githubusercontent.com/16143771/149631791-8cdc67aa-ca33-48d6-88b5-e69c1e03b873.png)